packages/application/src/shell.ts

```diff
@@ -213,12 +234,34 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
 
     leftHandler.sideBar.addClass(SIDEBAR_CLASS);
     leftHandler.sideBar.addClass('jp-mod-left');
+    leftHandler.sideBar.node.setAttribute('role', 'complementary');
+    leftHandler.sideBar.node.setAttribute(
+      'aria-label',
+      trans.__('main sidebar')
+    );
+    leftHandler.sideBar.contentNode.setAttribute('role', 'navigation');
+    leftHandler.sideBar.contentNode.setAttribute(
+      'aria-label',
+      trans.__('main sidebar navigation')
```

Comment:
Same thing for this one. My apologies - could have labeled my pictures better.
```suggestion
      trans.__('main sidebar')
```
![main sidebar double nav](https://user-images.githubusercontent.com/51969207/104764978-2412e280-5736-11eb-845f-0b33b60d3aa5.PNG)